src/hotspot/share/jfr/recorder/checkpoint/types/traceid/jfrTraceIdLoadBarrier.inline.hpp

```diff
@@ -87,7 +87,9 @@ inline traceid JfrTraceIdLoadBarrier::load(const Klass* klass) {
 }
 
 inline traceid JfrTraceIdLoadBarrier::load(const Method* method) {
-  return load(method->method_holder(), method);
+  auto holder = method->method_holder();
+  auto res = load(holder, method);
+  return res;
```

Comment:
Why this change?

Reply:
Good catch, I'll remove it

Reply:
I'll be cleaning up the changes tomorrow, I found two more.